scripts/import-styles/main.ts

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env -S deno run -A
+import usercssMeta from "usercss-meta";
+import { ensureDir } from "std/fs/mod.ts";
+import { walk } from "std/fs/walk.ts";
+import { parse as parseFlags } from "std/flags/mod.ts";
+import { join } from "std/path/mod.ts";
+
+import { REPO_ROOT } from "@/deps.ts";
+
+const flags = parseFlags(Deno.args, { boolean: ["fix"] });
+const subDir = flags._[0]?.toString() ?? "";
+const stylesheets = walk(join(REPO_ROOT, "styles", subDir), {
+  includeFiles: true,
+  includeDirs: false,
+  includeSymlinks: false,
+  match: [/\.user.css$/],
+});
+
+// Recommended settings.
+const settings = {
+  settings: {
+    updateInterval: 24,
+    updateOnlyEnabled: true,
+    patchCsp: true,
+  },
+};
+
+const data = [settings];
```

Comment:
Just some typing since my local deno lint complains about `TS2345` here
```suggestion
const data: Record<string, unknown>[] = [settings];
```

```console
Check file:///Users/winston/Code/catppuccin/userstyles/scripts/import-styles/main.ts
error: TS2345 [ERROR]: Argument of type '{ enabled: boolean; name: any; description: any; author: any; url: any; updateUrl: any; usercssData: any; sourceCode: string; }' is not assignable to parameter of type '{ settings: { updateInterval: number; updateOnlyEnabled: boolean; patchCsp: boolean; }; }'.
  Object literal may only specify known properties, and 'enabled' does not exist in type '{ settings: { updateInterval: number; updateOnlyEnabled: boolean; patchCsp: boolean; }; }'.
    enabled: true,
```